src/routes/index.tsx

```diff
@@ -87,7 +87,9 @@ const Routes: React.FunctionComponent<{}> = props => {
         />
         <Route
           path={'/pcrs'}
-          component={requiresIdentity(DelegationsView, { isPCR: true })}
```

Comment:
Change request:
In general, if your ticket doesn't affect a file functionally, the file shouldn't be part of your PR. It makes it easier for you and your reviewer!

Reply:
Ok, I think my Pretty is active again. Must have noticed the needed change.

Reply:
I have updated it.